azure-pipelines.yml

```diff
@@ -41,6 +41,15 @@ steps:
       npm run perf-test
     displayName: perf test
 
+  - task: GithubPRComment@0
+    displayName: 'Post Perf Results to Github Pull Request'
+    inputs:
+      githubOwner: OfficeDev
+      githubRepo: 'office-ui-fabric-react'
+      blobFilePath: '$(Build.SourcesDirectory)/$(PerfCommentFilePath)'
+      status: '$(PerfCommentStatus)'
+      uniqueId: 'perfComment9423'
```

Comment:
What's the thinking behind uniqueId in the yaml file?

Reply:
uniqueId is used by the commentPr task to identify comments posted by the job. If uniqueId specified, the comment is updated instead of created anew.

Reply:
Gotcha! :+1: Can we add a comment here though explaining this for the sake of future maintainers?